components/todo-list.js

```diff
@@ -3,17 +3,17 @@ import styled from "styled-components";
 
 import TodoItem from "./todo-item";
 
-export default function TodoList() {
-  return (
-    <Wrapper>
-      Todo list가 들어가는 자리입니다!
-      <TodoItem />
-    </Wrapper>
-  );
+export default function TodoList({ todos, completeTodo }) {
+    return (
+        <Wrapper>
+            {todos.sort((todo1, todo2) => {
+                return todo1.dueDate < todo2.dueDate ? -1 : todo1.dueDate > todo2.dueDate ? 1 : 0;
```

Comment:
sort의 compare 함수는 양수, 0, 음수여부만 판단하기 때문에 
```suggestion
                return todo1.dueDate - todo2.dueDate;
```
라고만 해주셔도 됩니다 ㅎ